crates/uv-toolchain/src/discovery.rs

```diff
@@ -115,29 +116,12 @@ type ToolchainResult = Result<Toolchain, ToolchainNotFound>;
 
 /// The result of failed toolchain discovery.
 ///
-/// See [`InterpreterResult`].
+/// See [`ToolchainResult`].
 #[derive(Clone, Debug, Error)]
-pub enum ToolchainNotFound {
```

Comment:
The enum is now a struct which contains the `ToolchainRequest` instead of de-structuring the request here.